app/controllers/admin/auditor_controller.rb

```diff
@@ -149,15 +149,14 @@ def get_account_filter
 
   def get_report
     report = nil
-    @account_filter = get_account_filter unless @account_filter.blank?
     if params[:report]
       report = ReportArchive.where(id: params[:report]).first
       params.delete :report
     else
       #start by saving the report (add check to see if there is a report)
       reports = ReportArchive.where(organization_id: @org.id) 
-      if !reports.blank?
-          report = reports.count == 1
+      if reports.present?
+        report = reports.first
```

Comment:
I still find this odd. 
`report = ReportArchive.find_by(organization_id: @org.id)` 
would do the same thing, with fewer lines of code. 
and if your associations are defined properly in the model you could even 
`report = ReportArchive.find_by(organization: @org)` 
which is the way I prefer to write it unless using the ID is necessary.